server.js

```diff
@@ -38,28 +36,117 @@ var renderHtml = function(url, cb) {
             }, 10000); //If no callPhantom script was called on the page, then wait 10secs and load page anyway.
         });
     };
+
+    console.debug("Opening page");
     page.open(url);
 
     page.onError = function (msg, trace) {
-        console.log(msg);
+        console.error(msg);
         trace.forEach(function(item) {
-            console.log('  ', item.file, ':', item.line);
+            console.error('  ', item.file, ':', item.line);
```

Comment:
That's a good approach to send different kinds of messages. But what's the difference for my real console output?